akka-cluster-bootstrap/src/main/resources/rp.conf

```diff
@@ -1,23 +1,27 @@
-# This needs to be in application.conf because reference.conf files have no guarantee
-# and Akka's values may override ours. If we were strictly in a Play app, we could use
-# reference-overrides.conf but as Akka Cluster also can be used standalone, here we are.
-
 akka {
```

Comment:
I'd give this file a better name, such as `reactive-lib-defaults.conf`, and describe how it's intended to be used (I guess it's intended to be included from the application's conf.